### ISSUE TITLE
Includes concepts that haven't been taught

### ISSUE THREAD

**Issue by jmburges** [learn-co-curriculum/flatiron-bnb-methods#9]
The Flatiron BNB Methods lab was difficult. The actual Ruby code was challenging, but doable. What was confusing were the associations. I know it's good to get stretched and challenged, but at the same time, I wish there was a more comprehensive lesson on these types of associations. I totally get that we need to supplement lessons with our own outside sources, but it almost feels as if the reading on this isn't sufficient. If the lab is going to test things like this: 

```
has_many :trip_listings, :through => :trips, :source => :listing
has_many :hosts, :through => :trip_listings, :foreign_key => 'host_id' has_many :host_reviews, :through => :guests, :source => :reviews
```

then there should at least be some information about sources and creating names like :trip_listings in a previous lesson, right? I spent A LOT of time doing outside reading on this. It seems really important to understand this.

**Comment by jmburges** [comment]
I want them to have to google to find the new ways to solve this. In this lab lets give them a hint that we meant for them to google, instead of just assuming they'll do it. Also maybe a few hints

**Comment by AnnJohn** [comment]
@pletcher : the first comment was what joe copied and pasted from a student. The second comment is what he feels the resolution to this issue should be.

**Comment by pletcher** [comment]
Closed by 37d7b0f
